internal/pkg/levee/levee.go

```diff
@@ -41,32 +41,29 @@ func run(pass *analysis.Pass) (interface{}, error) {
 	if err != nil {
 		return nil, err
 	}
-	sourcesMap := pass.ResultOf[source.Analyzer].(source.ResultType)
+	fnSources := pass.ResultOf[source.Analyzer].(source.ResultType)
```

Comment:
Does fn mean function here?

Reply:
Yes. Renamed to `funcSources` for clarity.